tests/features/native_on_error.feature

```diff
@@ -23,3 +23,10 @@ Feature: Native on error callbacks are invoked
     And I wait to receive a request
     And the request payload contains a completed unhandled native report
     And the event "context" equals "Some custom context"
+
+  Scenario: on_error returning false is removed
+    When I run "CXXRemoveOnErrorScenario" and relaunch the app
```

Comment:
This scenario does include an onError returning false, but did you intend to bundle separate scenarios in together?

Reply:
As above, I think we can remove some ambiguity from this by simplifying it to just modifying data.